### DISCUSSION TITLE
Add scrolls bar

### DISCUSSION THREAD

**Discussion by qadeer819** [Taewan-P/gpt_mobile#53] [Ideas]
Will be use full to have scrolls bars as it’s hard to figure out if at last prompts and make scrolls automatically to bottom response as have to scroll to end on every time open chat on Samsung a52

**Comment by qadeer819** [comment]
Will be nice if can upload files and photos etc

**Comment by Taewan-P** [comment]
Scroll to the top or bottom floating button may be implemented later. No scrollbars though

Multimodal is planned (See README.md)

**Comment by qadeer819** [comment]
Let’s assume you are working with two chats at the same time. Every time Chat 1 is opened, it starts from the top, and Chat 2 starts from the top as well. Each time you switch between them, they start from the top because the main important parts are the latest replies, which are at the bottom. Ideally, every chat should open on the last output or prompt, and there should be no scrolling or swiping through blank white spaces. When you try to slide down, it goes way past the last output, and there is no real track of the conversation since there is a lot of padding between prompts and outputs. This makes it hard to tell if you are scrolling through blank space or if there is more output.

**Comment by Taewan-P** [comment]
Hello, this feature is now implemented in [v0.7.0](https://github.com/Taewan-P/gpt_mobile/releases). Thanks!
